### ISSUE TITLE
CLI Git Release prefix breaks releases API

### ISSUE THREAD

**Issue by coolaj86** [watchexec/watchexec#225]
Hello!

I see that you added "CLI" as a prefix to the version because now both CLI and Lib and released.

However, this breaks the use of the GitHub releases API, which is generally expected to be a semver tag for the CLI binary.

Would you consider removing the CLI prefix so that the binaries can be handled the same way that they were before, and the way that other binaries are issued on GitHub Releases?

Thankfully it didn't break-break webinstall.dev, but it did cause the download to have a funny name:

<img width="195" alt="Screen Shot 2021-11-15 at 1 38 37 AM" src="https://user-images.githubusercontent.com/122831/141749329-5947ef30-edca-4569-ad63-327558b503b3.png">

**Comment by passcod** [comment]
There's no mention of semver at all in the gh release API docs. It doesn't break the use of the API, it breaks "whatever thing was using it"'s faulty assumptions that all releases are purely semver. A scheme with versions prefixed with what is being versioned is actually fairly common with monorepos, which this now qualifies as. Some of these monorepos have a primary unprefixed version, but it's not necessarily the CLI. Indeed, often it's for the main library, with the CLI separate.

Additionally, it's... been a while since that changed. I don't want to break every _other_ downstream repackager just because of one that can't deal.

TL;DR I'm not going to change a perfectly good scheme with a worse, more ambiguous one because a downstream has a bug.
